src/command.ts

```diff
@@ -142,7 +142,9 @@ export function command<
           continue;
         }
 
-        if (node.type === 'shortOptions') {
+        if (node.type === 'forcePositional') {
+          continue;
```

Comment:
Can you add a comment here that explains why we just do the `continue`?
```suggestion
          // A `forcePositional` node can't really be visited since it has no meaning
          // other than forcing a positional argument in the parsing phase
          continue;
```